dashboard/db/migrate/20170502231246_create_courses.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCourses < ActiveRecord::Migration[5.0]
+  def change
+    create_table :courses do |t|
+      t.string :name
+      t.integer :plc_course_id
```

Comment:
Ah sorry, I was thinking this was the primary key for this table.

It looks like you have the associate set up to map `Plc::Course` -> `Course` (which makes sense to me because not every `Course` will have a `Plc::Course` but every `Plc::Course` should have an associated `Course`).